llvm/test/CodeGen/RISCV/fp-maximumnum-minimumnum.ll

```diff
@@ -0,0 +1,734 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
```

Comment:
Can you add these tests to the files we already test llvm.maxnum/minnum in like `float-intrinsics.ll?

Reply:
It is OK for me, while I thought that this case is some big.
I will migrate these cases to float-intrinsics.ll.